.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I think you didn't mean to commit this

Reply:
@uramirez8707 
Done. Thanks.
We shoudl consider putting a good one in.